CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- mwc-button
```

Comment:
Mind matching the format of what's below? I was doing Changed/Added/Fixed sections. Also a little more detail would be nice. Why would a user care about new focus and blur functions?